server/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const { Stay, db } = require('../database/index');
+
+const app = express();
+const port = 3000;
+
+app.get('/stays', (req, res) => {
+  Stay.find({}, (err, data) => {
+    if (err) {
+      res.status(400).send('Error Find All Failed.');
```

Comment:
May be able to send 'err' instead of hard coded error

Reply:
Fixed